source/administration/config-settings.rst

```diff
@@ -223,6 +227,13 @@ Maximum number of channels per team, including both active and deleted channels.
 | This feature's ``config.json`` setting is ``"MaxChannelsPerTeam": 2000`` with whole number input.                                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Max Notifications Per Channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
+Maximum total number of users in a channel before users typing messages, @all, @here, and @channel no longer send notifications because of performance.
```

Comment:
Propose `... before @all, @here or @channel no longer send notifications to maximize performance.`